File: server/router/yunguan/config/licenseConfig.go
package config

import (
	"github.com/gin-gonic/gin"
	v1 "ygpt/server/api/v1"
)

type LicenseConfigRouter struct {
}

func (s *LicenseConfigRouter) InitLicenseConfigRouter(Router *gin.RouterGroup) {
	dataRouter := Router.Group("config/licenseConfig")
	backApi := v1.ApiGroupApp.YunGuanApiGroup.ConfigApi.LicenseConfigApi

	{
		dataRouter.POST("updateLicense", backApi.UpdateLicenseConfig) // 添加数据
		dataRouter.POST("queryLicense", backApi.GetLicenseConfig)     // 修改数据
		dataRouter.POST("loginLicense", backApi.LoginLicenseConfig)   // 修改数据
		dataRouter.POST("updateLicensePwd", backApi.UpdateLicensePwd) // 修改数据
	}
}

File: web/src/api/yunguan/config/licenseConfig.js
import request from '@/utils/request'
// 添加元素
export function updateLicense(modelData) {
    return request({
        url: 'config/licenseConfig/updateLicense',
        method: 'post',
        data: modelData
    })
}
// 添加元素
export function updateLicensePwd(modelData) {
    return request({
        url: 'config/licenseConfig/updateLicensePwd',
        method: 'post',
        data: modelData
    })
}
// 添加元素
export function queryLicense(modelData) {
    return request({
        url: 'config/licenseConfig/queryLicense',
        method: 'post',
        data: modelData
    })
}
// 添加元素
export function loginLicense(modelData) {
    return request({
        url: 'config/licenseConfig/loginLicense',
        method: 'post',
        data: modelData
    })
}